dot/parachain/prospective-parachains/prospective_parachains.go

```diff
@@ -14,7 +14,10 @@ import (
 	"github.com/ChainSafe/gossamer/lib/common"
 )
 
-var logger = log.NewFromGlobal(log.AddContext("pkg", "parachain-prospective-parachains"), log.SetLevel(log.Debug))
+var logger = log.NewFromGlobal(
+	log.AddContext("pkg", "parachain-prospective-parachains"),
+	log.SetLevel(log.Debug),
```

Comment:
Shouldn't the log level be set in config/CLI flags?

Reply:
yeah, removing this configuration